lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -245,12 +245,7 @@ namespace
         MString uvSetName(primvarName.GetText());
         bool createUVSet = true;
 
-        if (primvarName == UsdUtilsGetPrimaryUVSetName()) {
-            // We assume that the primary USD UV set maps to Maya's default 'map1'
-            // set which always exists, so we shouldn't try to create it.
-            uvSetName = UsdMayaMeshPrimvarTokens->DefaultMayaTexcoordName.GetText();
-            createUVSet = false;
-        } else if (!hasDefaultUVSet) {
+        if (!hasDefaultUVSet) {
```

Comment:
The removes the automatic "st"-> "map1" renaming on import. Should I create an env var for shops that have internal tools that depend on this behavior?

Reply:
Yes, please! Export is more of a concern for us than import (more detail on that below), but it would be good to have a `TfEnvSetting` we could set for each direction.